feat(content): Korath Expansion Stage II - World-Ships (#7088)
endless-sky/endless-sky: data/korath/korath ships.txt

		"cargo space" 1052
		"outfit space" 1038
		"weapon capacity" 331
		"engine capacity" 251
		weapon
			"blast radius" 400
			"shield damage" 8000
			"hull damage" 4000
			"hit force" 12000
	outfits
		"Korath Banisher" 3
		"Korath Warder" 3
		"Korath Grab-Strike" 2
		
		"Triple Plasma Core"
		"Systems Core (Large)"
		"Large Heat Shunt" 2
		"Fuel Processor" 2
		"Brig"
		"Luxury Accommodations"
		"Microbot Defense Station" 23
		"Korath Repeater Rifle" 372
		
		"Afterburner (Comet Class)"
		"Reverser (Lunar Class)"
		"Thruster (Stellar Class)"
		"Steering (Planetary Class)"
		"Scram Drive"
	
	engine -37 343 .8
	engine 37 343 .8
	"reverse engine" -16 -336 0.9
	"reverse engine" 16 -336 0.9
	"reverse engine" 0 -303 0.5
		over
	turret -78 -233.5 "Korath Banisher"
	turret 78 -233.5 "Korath Warder"
	turret -78 9.5 "Korath Grab-Strike"
	turret 78 9.5 "Korath Grab-Strike"
	turret -73.5 41 "Korath Banisher"
	turret 73.5 41 "Korath Warder"
	turret -78 231.5 "Korath Warder"
	turret 78 231.5 "Korath Banisher"
	explode "tiny explosion" 30
	explode "small explosion" 45
	explode "medium explosion" 50
	explode "large explosion" 40
	explode "huge explosion" 50
	"final explode" "final explosion large"

> 	description "When the Korath were exiled, it was on these gargantuan 'world-ships' they first fled and lived on. Since the introduction of purpose-built city ships, these ancient vessels have mostly been decommissioned, turned into stations, or reconfigured into smaller, specialized vessels. Yet a few still ply the starlanes, crewed by those too restless to live on stations yet not bold enough to go on raids."

Author: ```suggestion
	description "Originally designed as a a luxury habitat for those wealthy enough to escape living on the polluted surface of Korath worlds, when the Korath were exiled, it was on these gargantuan 'world-ships' they first crammed themselves into and fled to the core.  Nostalgically, the class was then renamed after the devastated and now forbidden Korath homeworld."
	description "
	Since the introduction of purpose-built city ships, these ancient vessels have mostly been decommissioned, turned into stations, or reconfigured into smaller, specialized vessels. Yet a few still ply the starlanes, crewed by those too restless to live on stations yet not bold enough to go on raids."
```

		angle 70
	bay "Fighter" -167.5 -420
		angle 315
	bay "Fighter" 167.5 -420
		angle 45
	bay "Fighter" -179.5 80
		angle 45
		back
	bay "Fighter" 179.5 80
		angle 315
		back
	bay "Fighter" -167.5 -420
		angle 315
	bay "Fighter" 167.5 -420
		angle 45
	bay "Fighter" -179.5 80
		angle 45
		back
	bay "Fighter" 179.5 80
		angle 315
		back
	bay "Fighter" -167.5 -420
		angle 315
	bay "Fighter" 167.5 -420
		angle 45
	bay "Fighter" -179.5 80
		angle 45
		back
	bay "Fighter" 179.5 80
		angle 315
		back
	bay "Fighter" -167.5 -420
		angle 315
	bay "Fighter" 167.5 -420
		angle 45
	bay "Fighter" -179.5 80
		angle 45
		back
	bay "Fighter" 179.5 80
		angle 315
		back
	bay "Fighter" -167.5 -420
		angle 315
	bay "Fighter" 167.5 -420
		angle 45
	bay "Fighter" -179.5 80
		angle 45
		back
	bay "Fighter" 179.5 80
		angle 315
		back
	bay "Fighter" -167.5 -420
		angle 315
	bay "Fighter" 167.5 -420
		angle 45
	bay "Fighter" -179.5 80
		angle 45
		back
	bay "Fighter" 179.5 80
		angle 315
		back
	explode "tiny explosion" 30
	explode "small explosion" 45
	explode "medium explosion" 50
	explode "large explosion" 40
	explode "huge explosion" 50
	"final explode" "final explosion large"

> 	description "Korsmanath A'awojs are massive nomadic habitats that hold much of their remaining population.  True flying cities, they could be self-contained miniature worlds were it not for their heavy defensive armament and reliance on unsustainable materials, methods, and sources of power."

Author: ```suggestion
	description "Korsmanath A'awojs, named in memory of the Korath home system, are massive nomadic habitats that hold much of their remaining population.  True flying cities, they could be self-contained miniature worlds were it not for their heavy defensive armament and reliance on unsustainable materials, methods, and sources of power."
```

> 	description "  Commission of new Havens is an intensely resource-demanding, labour-intensive, painstaking process fraught with political intrigue. Navarchs fight with the civilian government for resources and yard time they would rather devote to building raiding and defensive fleets, civilians vie with each other for a place on the next ship, and planning committees argue with engineers over what methods and technologies to use. This leads to construction delays, corner-cutting, and over-militarisation of the design at best, and violent feuds at worst."

Reviewer: I don't recall the term Navarch being used anywhere in the game besides here. Will it be made clear what a Navarch is to the player in future content?

Author: Yes, the lore we've been developing is that the Exiles - while having a *de jure* civilian government - are in many areas dominated rather by a military junta of "Navarchs," that is, admiral-rulers.  We do plan to expand upon this in future missions.

	fighter 98 156 back
	fighter -50 303 back
	fighter -50 303 back
	fighter 50 303 back
	fighter 50 303 back
	drone -100 -169
	drone 100 -169
	drone -98 156 back
	drone 98 156 back
	drone -50 303 back
	drone 50 303 back
	gun -71 -209 "Korath Fire-Lance"
	gun 0 -209 "Korath Fire-Lance"
	gun 71 -209 "Korath Fire-Lance"
	turret -52 -295 "Korath Banisher"
	turret 52 -295 "Korath Warder"
	turret -104 -13 "Korath Grab-Strike"
	turret 0 -13 "Korath Grab-Strike"
	turret 104 -13 "Korath Grab-Strike"
	turret -80.5 191.5 "Korath Warder"
	turret 80.5 191.5 "Korath Banisher"
	explode "tiny explosion" 30
	explode "small explosion" 45
	explode "medium explosion" 50
	explode "large explosion" 40
	explode "huge explosion" 50
	"final explode" "final explosion large"
	description "The largest of Korath production vessels are derived from a common and ancient chassis. This many-lobed ship has been adapted to carry fighters into large-scale raids, or mining craft into asteroid belts. Its radial arrangements of hangars allows fighters a maximum variety of safe approach patterns in combat."

ship "Kasichara A'awoj"
	sprite "ship/world-ship f"
	thumbnail "thumbnail/world-ship f"
	attributes

> 		category "Space Liner"

Reviewer 2: This really seems like it belongs in the Utility category.

Author: I rather like the symmetry of two liners, two warships, and two freighters.... and this is the least well-armed of the six..

but, i suppose the crew:cargo ratio is a bit low

could raise crew, reduce cargo.... but, while these will be generally uncapturable, we might want one to show up in a climactic mission or *maybe* two, and it's already a *lot* of bunks even for a trophy

		angle 315
		back
	bay "Fighter" -167.5 -420
		angle 315
	bay "Fighter" 167.5 -420
		angle 45
	bay "Fighter" -179.5 80
		angle 45
		back
	bay "Fighter" 179.5 80
		angle 315
		back
	bay "Fighter" -167.5 -420
		angle 315
	bay "Fighter" 167.5 -420
		angle 45
	bay "Fighter" -179.5 80
		angle 45
		back
	bay "Fighter" 179.5 80
		angle 315
		back
	bay "Fighter" -167.5 -420
		angle 315
	bay "Fighter" 167.5 -420
		angle 45
	bay "Fighter" -179.5 80
		angle 45
		back
	bay "Fighter" 179.5 80
		angle 315
		back
	bay "Fighter" -167.5 -420
		angle 315
	bay "Fighter" 167.5 -420
		angle 45
	bay "Fighter" -179.5 80
		angle 45
		back
	bay "Fighter" 179.5 80
		angle 315
		back
	explode "tiny explosion" 30
	explode "small explosion" 45
	explode "medium explosion" 50
	explode "large explosion" 40
	explode "huge explosion" 50
	"final explode" "final explosion large"
	description "Korsmanath A'awojs, named in memory of the Korath home system, are massive nomadic habitats that hold much of their remaining population.  True flying cities, they could be self-contained miniature worlds were it not for their heavy defensive armament and reliance on unsustainable materials, methods, and sources of power."

> 	description "  Commission of new Havens is an intensely resource-demanding, labour-intensive, painstaking process fraught with political intrigue. Navarchs fight with the civilian government for resources and yard time they would rather devote to building raiding and defensive fleets, civilians vie with each other for a place on the next ship, and planning committees argue with engineers over what methods and technologies to use. This leads to construction delays, corner-cutting, and over-militarisation of the design at best, and violent feuds at worst."

Author: Ack you caught me :-P
~~for my sanity's sake we really need an en-gb release~~

	explode "tiny explosion" 30
	explode "small explosion" 45
	explode "medium explosion" 50
	explode "large explosion" 40
	explode "huge explosion" 50
	"final explode" "final explosion large"
	description "The largest of Korath production vessels are derived from a common and ancient chassis. This simple ship has been adapted for hauling materials between refinery and factory stations."

ship "Ra'gru Ik 618"
	sprite "ship/world-ship b"
	thumbnail "thumbnail/world-ship b"
	attributes
		category "Heavy Warship"
		cost 37640000
		shields 51000
		hull 39500
		"required crew" 619
		"bunks" 794
		"mass" 1735
		"drag" 21
		"heat dissipation" .48
		"fuel capacity" 1000
		"cargo space" 191
		"outfit space" 1278
		"weapon capacity" 633
		"engine capacity" 263
		weapon
			"blast radius" 400
			"shield damage" 8000
			"hull damage" 4000
			"hit force" 12000
	outfits
		"Grab-Strike Turret" 2
		"Banisher Grav-Turret" 2
		"Warder Anti-Missile" 2
		"Fire-Lance" 3
		"Korath Inferno" 2
		"Blaze-Pike Turret" 2
		"Firelight Missile Bank" 2
		"Firelight Missile" 36
		
		"Triple Plasma Core"
		"Systems Core (Large)"
		"Large Heat Shunt" 2
		"Fuel Processor"
		"Brig"
		"Microbot Defense Station" 23

> 		"Thermal Repeater Rifle" 359

Reviewer 3: ```suggestion
		"Thermal Repeater Rifle" 794
```

Given that this is the warship variant, and specifically built for combat and potentially raiding, I think it would make sense for it to have an offensive rifle for every required crew, with extras for temporary crew.

	turret 80 283 "Warder Anti-Missile"
	explode "tiny explosion" 30
	explode "small explosion" 45
	explode "medium explosion" 50
	explode "large explosion" 40
	explode "huge explosion" 50
	"final explode" "final explosion large"
	description "The largest of Korath production vessels are derived from a common and ancient chassis. This relatively streamlined ship has been adapted to collect, refine, and transport hyperspace fuel."

ship "Lor'nag Ik 590"
	sprite "ship/world-ship d"
	thumbnail "thumbnail/world-ship d"
	attributes
		category "Space Liner"
		cost 29640000
		shields 47000
		hull 34800
		"required crew" 538
		"bunks" 1492
		"mass" 1900
		"drag" 18
		"heat dissipation" .37
		"solar collection" 5.6
		"fuel capacity" 1000
		"cargo space" 264
		"outfit space" 856
		"weapon capacity" 321
		"engine capacity" 241
		weapon
			"blast radius" 400
			"shield damage" 8000
			"hull damage" 4000
			"hit force" 12000
	outfits
		"Shunt-Strike Turret" 2
		"Banisher Grav-Turret" 2
		"Warder Anti-Missile" 2
		"Fire-Lance" 2
		
		"Triple Plasma Core"
		"Systems Core (Large)"
		"Large Heat Shunt" 2
		"Fuel Processor"
		"Brig"
		"Microbot Defense Station" 15

> 		"Thermal Repeater Rifle" 272

Reviewer 3: ```suggestion
		"Thermal Repeater Rifle" 1492
```

Given that, when we run into these, we don't know if they're on transport duty or just on their too or from a raid, or anything; I'm inclined to just have them carry a full complement as if they were on raiding duty. That being said, given the prevalence of warfare in the Korath territories; it would be believable to me that they would have a rifle for every person on board a military transport even if it *was* just for transport duty. Never know when they might need to board someone (or fend off boarders, for that matter)..

Author: I think it would be better to set up two variants - one as is, with 272 rifles and 15 microbot stations as it is here, for civilian use, and a second with 1492 rifles and perhaps not using as many microbots if needed for combat outfits, for troop transport use.

Reviewer 3: Sure, that could work for civilians. In general, though, there's really no way to tell if it's just transporting people, or soldiers, or going to a raid, or whatever. (and the way our boarding values are setup, all Korath are equally tough. (Just like all Quarg are equally good at combat. And all arach, kimek, etc.) So from a practical gameplay perspective, there's no difference between military and civilians as far as boarding combat goes.

Author: On the contrary, we *do* know.  This ship appears in at least two fleets already, each with different roles and behaviours.  And then there's mission appearances which can pick their own variants.

Author: We do though.  There are three variants of the Lor'nag Ik 590: the base, a grab-strike variant, and an explicit troop transport version not yet used but set aside for mission use.

I've gone over these now -- setting up variants is one of those tasks I tend to delegate -- and made some edits so the usage is more consistent.
 - Base 590s with minimal, defensive loadout only appear in Civilian fleets: 272 rifles, 17 microbots
 - Military 590s with Grab-Strikes only appear in hostile home fleets: 493 rifles, 15 microbots
 - Troop 590s with Shunt-Strike do not appear yet, but should take the place of a couple ground assault dredgers in missions. 1492 rifles, 8 mircobots

		hull 29600
		"required crew" 234
		"bunks" 453
		"mass" 1915
		"drag" 19
		"heat dissipation" .4
		"fuel capacity" 1000
		"cargo space" 1375
		"outfit space" 883
		"weapon capacity" 338
		"engine capacity" 237
		weapon
			"blast radius" 400
			"shield damage" 8000
			"hull damage" 4000
			"hit force" 12000
	outfits
		"Banisher Grav-Turret" 2
		"Warder Anti-Missile" 4
		"Grab-Strike Turret" 2
		"Expeller Grav-Ray" 2
		
		"Triple Plasma Core"
		"Systems Core (Medium)"
		"Large Heat Shunt" 2
		"Fuel Processor"
		"Thermal Repeater Rifle" 237
		"Microbot Defense Station" 17
		
		"Thruster (Stellar Class)"
		"Steering (Planetary Class)"
		"Scram Drive"
	
	engine -37 333 .8
	engine 0 333 .8
	engine 37 333 .8
	"reverse engine" -19 -303 0.5
		over
	"reverse engine" 0 -303 0.5
		over
	"reverse engine" 19 -303 0.5
		over
	gun -47 -244 "Expeller Grav-Ray"
	gun 47 -244 "Expeller Grav-Ray"
	turret -73 -159 "Warder Anti-Missile"
	turret 73 -159 "Grab-Strike Turret"
	turret -56 -137 "Warder Anti-Missile"
	turret 56 -137 "Warder Anti-Missile"
	turret -58 -6 "Banisher Grav-Turret"
	turret 58 -6 "Banisher Grav-Turret"
	turret -58 25 "Grab-Strike Turret"
	turret 58 25 "Warder Anti-Missile"

> 	fighter -72 294 back
> 	fighter 0 294 back
> 	fighter 72 294 back

Reviewer 4: ```suggestion
	bay "Fighter" -72 294 back
	bay "Fighter" 0 294 back
	bay "Fighter" 72 294 back
```

ship "Kas'lor Ik 582"
	sprite "ship/world-ship c"
	thumbnail "thumbnail/world-ship c"
	attributes
		category "Heavy Freighter"
		cost 27690000
		shields 39750
		hull 26100
		"required crew" 314
		"bunks" 551
		"mass" 1899
		"drag" 17
		"heat dissipation" .35
		"solar collection" 3.8
		"fuel capacity" 32700
		"cargo space" 734
		"outfit space" 839
		"weapon capacity" 321
		"engine capacity" 241
		weapon
			"blast radius" 400
			"shield damage" 8000
			"hull damage" 4000
			"hit force" 12000
	outfits
		"Langrage Hyper-Heaver" 2
		"Banisher Grav-Turret" 3
		"Warder Anti-Missile" 3
		
		"Triple Plasma Core"
		"Systems Core (Large)"
		"Large Heat Shunt" 2
		"Fuel Processor"
		"Brig"
		"Thermal Repeater Rifle" 272
		"Microbot Defense Station" 4
		
		"Thruster (Planetary Class)"
		"Steering (Planetary Class)"
		"Hyperdrive"
	
	engine -37 333 .8
	engine 0 333 .8
	engine 37 333 .8
	"reverse engine" -19 -303 0.5
		over
	"reverse engine" 0 -303 0.5
		over
	"reverse engine" 19 -303 0.5
		over

> 	fighter -70 51 back
> 	fighter 0 51 back
> 	fighter 70 51 back

Reviewer 4: ```suggestion
	bay "Fighter" -70 51 back
	bay "Fighter" 0 51 back
	bay "Fighter" 70 51 back
```

ship "Lor'nag Ik 590"
	sprite "ship/world-ship d"
	thumbnail "thumbnail/world-ship d"
	attributes
		category "Space Liner"
		cost 29640000
		shields 47000
		hull 34800
		"required crew" 538
		"bunks" 1492
		"mass" 1900
		"drag" 18
		"heat dissipation" .37
		"solar collection" 5.6
		"fuel capacity" 1000
		"cargo space" 264
		"outfit space" 856
		"weapon capacity" 321
		"engine capacity" 241
		weapon
			"blast radius" 400
			"shield damage" 8000
			"hull damage" 4000
			"hit force" 12000
	outfits
		"Grab-Strike Turret" 1
		"Banisher Grav-Turret" 3
		"Warder Anti-Missile" 2
		"Digger Mining Beam" 2
		
		"Triple Plasma Core"
		"Systems Core (Large)"
		"Large Heat Shunt" 2
		"Small Heat Shunt" 2
		"Fuel Processor"
		"Brig"
		"Microbot Defense Station" 17
		"Thermal Repeater Rifle" 272
		
		"Thruster (Stellar Class)"
		"Steering (Planetary Class)"
		"Scram Drive"
	
	engine -37 333 .8
	engine 0 333 .8
	engine 37 333 .8
	"reverse engine" -19 -303 0.5
		over
	"reverse engine" 0 -303 0.5
		over
	"reverse engine" 19 -303 0.5
		over

> 	fighter -90 160 back
> 	fighter 0 160 back
> 	fighter 90 160 back

Reviewer 4: ```suggestion
	bay "Fighter" -90 160 back
	bay "Fighter" 0 160 back
	bay "Fighter" 90 160 back
```

ship "Ra'at Ik 621"
	sprite "ship/world-ship e"
	thumbnail "thumbnail/world-ship e"
	attributes
		category "Heavy Warship"
		cost 33590000
		shields 48000
		hull 31400
		"required crew" 794
		"bunks" 1037
		"mass" 1762
		"drag" 27
		"heat dissipation" .41
		"fuel capacity" 1000
		"cargo space" 228
		"outfit space" 999
		"weapon capacity" 407
		"engine capacity" 277
		weapon
			"blast radius" 400
			"shield damage" 8000
			"hull damage" 4000
			"hit force" 12000
	outfits
		"Grab-Strike Turret" 3
		"Banisher Grav-Turret" 2
		"Warder Anti-Missile" 2
		"Fire-Lance" 3
		
		"Triple Plasma Core"
		"Systems Core (Large)"
		"Large Heat Shunt" 2
		"Fuel Processor"
		"Microbot Defense Station" 16
		"Thermal Repeater Rifle" 308
		
		"Reverser (Lunar Class)"
		"Thruster (Stellar Class)"
		"Steering (Stellar Class)"
		"Jump Drive"
	
	engine -42 343 .6
	engine -22 343 .6
	engine -22 343 .6
	engine 22 343 .6
	engine 22 343 .6
	engine 42 343 .6
	"reverse engine" -16 -336 0.9
	"reverse engine" 16 -336 0.9

> 	fighter -100 -169
> 	fighter 0 -169
> 	fighter 100 -169
> 	fighter -98 156 back
> 	fighter 98 156 back
> 	fighter -50 303 back
> 	fighter -50 303 back
> 	fighter 50 303 back
> 	fighter 50 303 back
> 	drone -100 -169
> 	drone 100 -169
> 	drone -98 156 back
> 	drone 98 156 back
> 	drone -50 303 back

Reviewer 4: ```suggestion
	bay "Fighter" -100 -169
	bay "Fighter" 0 -169
	bay "Fighter" 100 -169
	bay "Fighter" -98 156 back
	bay "Fighter" 98 156 back
	bay "Fighter" -50 303 back
	bay "Fighter" -50 303 back
	bay "Fighter" 50 303 back
	bay "Fighter" 50 303 back
	bay "Drone" -100 -169
	bay "Drone" 100 -169
	bay "Drone" -98 156 back
	bay "Drone" 98 156 back
	bay "Drone" -50 303 back
```
Swap for new keyword for bays, `back` probably should be on line down with another indent as well but haven't find way to type tabs on phone yet.

		angle 315
		back
	bay "Fighter" -167.5 -420
		angle 315
	bay "Fighter" 167.5 -420
		angle 45
	bay "Fighter" -179.5 80
		angle 45
		back
	bay "Fighter" 179.5 80
		angle 315
		back
	bay "Fighter" -167.5 -420
		angle 315
	bay "Fighter" 167.5 -420
		angle 45
	bay "Fighter" -179.5 80
		angle 45
		back
	bay "Fighter" 179.5 80
		angle 315
		back
	bay "Fighter" -167.5 -420
		angle 315
	bay "Fighter" 167.5 -420
		angle 45
	bay "Fighter" -179.5 80
		angle 45
		back
	bay "Fighter" 179.5 80
		angle 315
		back
	bay "Fighter" -167.5 -420
		angle 315
	bay "Fighter" 167.5 -420
		angle 45
	bay "Fighter" -179.5 80
		angle 45
		back
	bay "Fighter" 179.5 80
		angle 315
		back
	explode "tiny explosion" 30
	explode "small explosion" 45
	explode "medium explosion" 50
	explode "large explosion" 40
	explode "huge explosion" 50
	"final explode" "final explosion large"
	description "Korsmanath A'awojs, named in memory of the Korath home system, are massive nomadic habitats that hold much of their remaining population.  True flying cities, they could be self-contained miniature worlds were it not for their heavy defensive armament and reliance on unsustainable materials, methods, and sources of power."

> 	description "  Commission of new Havens is an intensely resource-demanding, labor-intensive, painstaking process fraught with political intrigue. Navarchs fight with the civilian government for resources and yard time they would rather devote to building raiding and defensive fleets, civilians vie with each other for a place on the next ship, and planning committees argue with engineers over what methods and technologies to use. This leads to construction delays, corner-cutting, and over-militarization of the design at best, and violent feuds at worst."

Reviewer: Should `Havens` here be replaced by` Korsmanath A'awojs`?

> 	description "  Commission of new Havens is an intensely resource-demanding, labor-intensive, painstaking process fraught with political intrigue. Navarchs fight with the civilian government for resources and yard time they would rather devote to building raiding and defensive fleets, civilians vie with each other for a place on the next ship, and planning committees argue with engineers over what methods and technologies to use. This leads to construction delays, corner-cutting, and over-militarization of the design at best, and violent feuds at worst."

Author: ```suggestion
	description "  Commission of new Korsmanath A'awojs is an intensely resource-demanding, labor-intensive, painstaking process fraught with political intrigue. Navarchs fight with the civilian government for resources and yard time they would rather devote to building raiding and defensive fleets, civilians vie with each other for a place on the next ship, and planning committees argue with engineers over what methods and technologies to use. This leads to construction delays, corner-cutting, and over-militarization of the design at best, and violent feuds at worst."
```

	bay "Fighter" 72 294 back
	explode "tiny explosion" 30
	explode "small explosion" 45
	explode "medium explosion" 50
	explode "large explosion" 40
	explode "huge explosion" 50
	"final explode" "final explosion large"
	description "The largest of Korath production vessels are derived from a common and ancient chassis. This simple ship has been adapted for hauling materials between refinery and factory stations."

ship "Ra'gru Ik 618"
	sprite "ship/world-ship b"
	thumbnail "thumbnail/world-ship b"
	attributes
		category "Heavy Warship"
		cost 49640000
		shields 51000
		hull 39500
		"required crew" 619
		"bunks" 794
		"mass" 1735
		"drag" 21
		"heat dissipation" .48
		"fuel capacity" 1000
		"cargo space" 191
		"outfit space" 1278
		"weapon capacity" 633
		"engine capacity" 263
		weapon
			"blast radius" 400
			"shield damage" 8000
			"hull damage" 4000
			"hit force" 12000
	outfits
		"Grab-Strike Turret" 2
		"Banisher Grav-Turret" 2
		"Warder Anti-Missile" 2
		"Fire-Lance" 3
		"Korath Inferno" 2
		"Blaze-Pike Turret" 2
		"Firelight Missile Bank" 2
		"Firelight Missile" 36
		
		"Triple Plasma Core"
		"Systems Core (Large)"
		"Large Heat Shunt" 2
		"Fuel Processor"

> 		"Brig"

Reviewer 2: ```suggestion
```

Reviewer 3: Korath might need to deal with prisoners now and then too. Perhaps criminals? In modern day navies many larger ships have brigs.

	explode "medium explosion" 50
	explode "large explosion" 40
	explode "huge explosion" 50
	"final explode" "final explosion large"
	description "The largest of Korath production vessels are derived from a common and ancient chassis. This many-lobed ship has been adapted to carry fighters into large-scale raids, or mining craft into asteroid belts. Its radial arrangements of hangars allows fighters a maximum variety of safe approach patterns in combat."

ship "Kasichara A'awoj"
	sprite "ship/world-ship f"
	thumbnail "thumbnail/world-ship f"
	attributes
		category "Space Liner"
		cost 42290000
		shields 44000
		hull 49600
		"required crew" 543
		"bunks" 1997
		"mass" 2105
		"drag" 24
		"heat dissipation" .43
		"solar collection" 12.7
		"fuel capacity" 600
		"cargo space" 532
		"outfit space" 1038
		"weapon capacity" 331
		"engine capacity" 251
		weapon
			"blast radius" 400
			"shield damage" 8000
			"hull damage" 4000
			"hit force" 12000
	outfits
		"Banisher Grav-Turret" 3
		"Warder Anti-Missile" 3
		"Grab-Strike Turret" 2
		
		"Triple Plasma Core"
		"Systems Core (Large)"
		"Systems Core (Small)"
		"Large Heat Shunt" 2
		"Small Heat Shunt" 3
		"Fuel Processor" 2

> 		"Luxury Accommodations"

Reviewer 2: Did they steal it from the Syndicate and the commander of one off these ships decided to have it installed or something?
```suggestion
```

Author: Luxury Accomodations don't say anything about being a Syndicate outfit.

	explode "small explosion" 45
	explode "medium explosion" 50
	explode "large explosion" 40
	explode "huge explosion" 50
	"final explode" "final explosion large"
	description `The largest of Korath production vessels are derived from a common and ancient chassis, including this large Efreti vessel. Facing very different challenges, and taking some inspiration from their Quarg protectors, the Efreti descendant of the "World-Ships" of old is lighter, faster, and more efficient, but less powerful than its Exile counterparts.`

ship "Arch-Carrack"
	sprite "ship/arch-carrack"
	thumbnail "thumbnail/arch-carrack"
	attributes
		category "Heavy Freighter"
		"cost" 15815000
		"shields" 21800
		"hull" 19300
		"required crew" 54
		"bunks" 97
		"mass" 1177
		"drag" 14.9
		"heat dissipation" .5
		"fuel capacity" 600
		"cargo space" 635
		"outfit space" 582
		"weapon capacity" 193
		"engine capacity" 147
		"disruption resistance" 0.24
		"disruption resistance energy" 2.4
		"disruption resistance heat" 2.4
		
		weapon
			"blast radius" 300
			"shield damage" 4600
			"hull damage" 2300
			"hit force" 5900
	outfits
		"Banisher Grav-Turret" 1
		"Digger Mining Beam" 2
		"Warder Anti-Missile" 2
		

> 		"Systems Core (Medium)"
> 		"Triple Plasma Core"
> 		"Large Heat Shunt" 2
> 		"Small Heat Shunt" 2

Reviewer 2: ```suggestion
		"Double Plasma Core"
		"Systems Core (Medium)"
		"Large Heat Shunt"
```

	explode "large explosion" 50
	explode "huge explosion" 15
	"final explode" "final explosion large"
	description "Although they do not effectively provide for their own defense, the Efreti remain responsible for the day-to-day needs of civilian and cargo transportation. While not as bulky or powerful as other Korath designs, this freighter is much faster at making runs back and forth between the Efreti's handful of remaining planets, while still holding enough defensive loadout to ward off the odd Mereti drone that slips past the Quarg."

ship "Charm-Shallop"
	sprite "ship/charm-shallop"
	thumbnail "thumbnail/charm-shallop"
	attributes
		category "Utility"
		"cost" 16570000
		"shields" 19700
		"hull" 15200
		"required crew" 65
		"bunks" 253
		"mass" 575
		"drag" 8.5
		"heat dissipation" .6
		"fuel capacity" 500
		"cargo space" 194
		"outfit space" 569
		"weapon capacity" 153
		"engine capacity" 171
		"disruption resistance" 0.2
		"disruption resistance energy" 2
		"disruption resistance heat" 2
		weapon
			"blast radius" 250
			"shield damage" 3600
			"hull damage" 1800
			"hit force" 5400
	outfits
		"Digger Mining Beam" 3
		"Warder Anti-Missile"
		"Banisher Grav-Turret"
		

> 		"Triple Plasma Core"
> 		"Systems Core (Medium)"
> 		"Large Heat Shunt" 2
> 		"Small Heat Shunt"

Reviewer 2: ```suggestion
		"Double Plasma Core"
		"Systems Core (Medium)"
		"Large Heat Shunt"
		"Small Heat Shunt"
```

	description "	Fighters do not come equipped with a hyperdrive. You cannot carry a fighter unless you have a ship in your fleet with a fighter bay."

ship "Lor'kas Ik 577"
	sprite "ship/world-ship a"
	thumbnail "thumbnail/world-ship a"
	attributes
		category "Heavy Freighter"

> 		cost 28650000

Reviewer 2: ```suggestion
		cost 24650000
```

		"Steering (Planetary Class)"
		"Scram Drive"
	
	engine -37 333 .8
	engine 0 333 .8
	engine 37 333 .8
	"reverse engine" -19 -303 0.5
		over
	"reverse engine" 0 -303 0.5
		over
	"reverse engine" 19 -303 0.5
		over
	gun -47 -244 "Expeller Grav-Ray"
	gun 47 -244 "Expeller Grav-Ray"
	turret -73 -159 "Warder Anti-Missile"
	turret 73 -159 "Grab-Strike Turret"
	turret -56 -137 "Warder Anti-Missile"
	turret 56 -137 "Warder Anti-Missile"
	turret -58 -6 "Banisher Grav-Turret"
	turret 58 -6 "Banisher Grav-Turret"
	turret -58 25 "Grab-Strike Turret"
	turret 58 25 "Warder Anti-Missile"
	bay "Fighter" -72 294 back
	bay "Fighter" 0 294 back
	bay "Fighter" 72 294 back
	explode "tiny explosion" 30
	explode "small explosion" 45
	explode "medium explosion" 50
	explode "large explosion" 40
	explode "huge explosion" 50
	"final explode" "final explosion large"
	description "The largest of Korath production vessels are derived from a common and ancient chassis. This simple ship has been adapted for hauling materials between refinery and factory stations."

ship "Ra'gru Ik 618"
	sprite "ship/world-ship b"
	thumbnail "thumbnail/world-ship b"
	attributes
		category "Heavy Warship"

> 		cost 37640000

Reviewer 2: ```suggestion
		cost 49640000
```

	"reverse engine" -19 -303 0.5
		over
	"reverse engine" 0 -303 0.5
		over
	"reverse engine" 19 -303 0.5
		over
	gun -75 -112 "Fire-Lance"
	gun 75 -112 "Fire-Lance"
	gun 0 -112 "Fire-Lance"
	gun 42 90 "Firelight Missile Bank"
		angle 110
	gun -42 90 "Firelight Missile Bank"
		angle 250
	turret -56 -254 "Warder Anti-Missile"
	turret 56 -254 "Banisher Grav-Turret"
	turret -73.5 -221 "Korath Inferno"
	turret 73.5 -221 "Korath Inferno"
	turret -56 -186.5 "Grab-Strike Turret"
	turret 56 -186.5 "Grab-Strike Turret"
	turret -83 163.5 "Blaze-Pike Turret"
	turret 83 163.5 "Blaze-Pike Turret"
	turret -68 195 "Banisher Grav-Turret"
	turret 68 195 "Warder Anti-Missile"
	explode "tiny explosion" 30
	explode "small explosion" 45
	explode "medium explosion" 50
	explode "large explosion" 40
	explode "huge explosion" 50
	"final explode" "final explosion large"
	description "The largest of Korath production vessels are derived from a common and ancient chassis. This deceptively spindly ship bristles with weapon emplacements for heavy raiding or defending other vessels."  

ship "Kas'lor Ik 582"
	sprite "ship/world-ship c"
	thumbnail "thumbnail/world-ship c"
	attributes
		category "Heavy Freighter"

> 		cost 27690000

Reviewer 2: ```suggestion
		cost 20690000
```

		"Microbot Defense Station" 4
		
		"Thruster (Planetary Class)"
		"Steering (Planetary Class)"
		"Hyperdrive"
	
	engine -37 333 .8
	engine 0 333 .8
	engine 37 333 .8
	"reverse engine" -19 -303 0.5
		over
	"reverse engine" 0 -303 0.5
		over
	"reverse engine" 19 -303 0.5
		over
	bay "Fighter" -70 51 back
	bay "Fighter" 0 51 back
	bay "Fighter" 70 51 back
	turret -52.5 -80.5 "Banisher Grav-Turret"
	turret 52.5 -80.5 "Warder Anti-Missile"
	turret -57 -50.5 "Warder Anti-Missile"
	turret 57.5 -50.5 "Banisher Grav-Turret"
	turret -56.5 55 "Langrage Hyper-Heaver"
	turret 56.5 55 "Langrage Hyper-Heaver"
	turret -80 283 "Banisher Grav-Turret"
	turret 80 283 "Warder Anti-Missile"
	explode "tiny explosion" 30
	explode "small explosion" 45
	explode "medium explosion" 50
	explode "large explosion" 40
	explode "huge explosion" 50
	"final explode" "final explosion large"
	description "The largest of Korath production vessels are derived from a common and ancient chassis. This relatively streamlined ship has been adapted to collect, refine, and transport hyperspace fuel."

ship "Lor'nag Ik 590"
	sprite "ship/world-ship d"
	thumbnail "thumbnail/world-ship d"
	attributes
		category "Space Liner"

> 		cost 29640000

Reviewer 2: ```suggestion
		cost 25640000
```

		"Thermal Repeater Rifle" 272
		
		"Thruster (Stellar Class)"
		"Steering (Planetary Class)"
		"Scram Drive"
	
	engine -37 333 .8
	engine 0 333 .8
	engine 37 333 .8
	"reverse engine" -19 -303 0.5
		over
	"reverse engine" 0 -303 0.5
		over
	"reverse engine" 19 -303 0.5
		over
	bay "Fighter" -90 160 back
	bay "Fighter" 0 160 back
	bay "Fighter" 90 160 back
	gun -45 -253 "Digger Mining Beam"
	gun 45 -253 "Digger Mining Beam"
	turret -77 -116.5 "Grab-Strike Turret"
	turret 77 -116.5 "Warder Anti-Missile"
	turret -87 -83 "Banisher Grav-Turret"
	turret 87 -83 "Banisher Grav-Turret"
	turret -51 276.5 "Warder Anti-Missile"
	turret 51 276.5 "Banisher Grav-Turret"
	explode "tiny explosion" 30
	explode "small explosion" 45
	explode "medium explosion" 50
	explode "large explosion" 40
	explode "huge explosion" 50
	"final explode" "final explosion large"
	description "The largest of Korath production vessels are derived from a common and ancient chassis. This bulbous ship has been adapted to carry passengers between stations, or boarding parties into enemy strongholds."

ship "Ra'at Ik 621"
	sprite "ship/world-ship e"
	thumbnail "thumbnail/world-ship e"
	attributes
		category "Heavy Warship"

> 		cost 33590000

Reviewer 2: ```suggestion
		cost 43590000
```

	turret 41 -69 "Warder Anti-Missile"
	turret -41.5 264 "Warder Anti-Missile"
	turret 41.5 264 "Banisher Grav-Turret"
	gun -75 -250 "Digger Mining Beam"
	gun 75 -250 "Digger Mining Beam"
	gun -60 -214.5 "Digger Mining Beam"
	gun 60 -214.5 "Digger Mining Beam"
	explode "tiny explosion" 30
	explode "small explosion" 45
	explode "medium explosion" 50
	explode "large explosion" 40
	explode "huge explosion" 50
	"final explode" "final explosion large"
	description `The largest of Korath production vessels are derived from a common and ancient chassis, including this large Efreti vessel. Facing very different challenges, and taking some inspiration from their Quarg protectors, the Efreti descendant of the "World-Ships" of old is lighter, faster, and more efficient, but less powerful than its Exile counterparts.`

ship "Arch-Carrack"
	sprite "ship/arch-carrack"
	thumbnail "thumbnail/arch-carrack"
	attributes
		category "Heavy Freighter"
		"cost" 29370000
		"shields" 21800
		"hull" 19300
		"required crew" 89
		"bunks" 158
		"mass" 927
		"drag" 14.9
		"heat dissipation" .5
		"fuel capacity" 600
		"cargo space" 635
		"outfit space" 582
		"weapon capacity" 193
		"engine capacity" 147

> 		"disruption resistance" 0.35
> 		"disruption resistance energy" 7
> 		"disruption resistance heat" 3.5

Reviewer 2: ```suggestion
		"disruption resistance" 0.24
		"disruption resistance energy" 2.4
		"disruption resistance heat" 2.4
```

		"Gaktem GP Hybrid Steering"
		"Hyperdrive"

	engine 19.5 189.5
	engine -19.5 189.5
	engine 46 170 0.7
	engine -46 170 0.7
	"reverse engine" 14.5 -187.5
	"reverse engine" -14.5 -187.5
	turret 0 -119.5 "Warder Anti-Missile"
	turret 0 42 "Banisher Grav-Turret"
	turret 0 138.5 "Warder Anti-Missile"
	gun -37.5 -187 "Digger Mining Beam"
	gun 37.5 -187 "Digger Mining Beam"
	explode "tiny explosion" 120
	explode "small explosion" 60
	explode "medium explosion" 70
	explode "large explosion" 50
	explode "huge explosion" 15
	"final explode" "final explosion large"
	description "Although they do not effectively provide for their own defense, the Efreti remain responsible for the day-to-day needs of civilian and cargo transportation. While not as bulky or powerful as other Korath designs, this freighter is much faster at making runs back and forth between the Efreti's handful of remaining planets, while still holding enough defensive loadout to ward off the odd Mereti drone that slips past the Quarg."

ship "Charm-Shallop"
	sprite "ship/charm-shallop"
	thumbnail "thumbnail/charm-shallop"
	attributes
		category "Utility"
		"cost" 16570000
		"shields" 19700
		"hull" 15200

> 		"required crew" 103
> 		"bunks" 281

Reviewer 2: ```suggestion
		"required crew" 43
		"bunks" 181
```

		"mass" 575
		"drag" 8.5
		"heat dissipation" .6
		"fuel capacity" 500
		"cargo space" 194
		"outfit space" 569
		"weapon capacity" 153
		"engine capacity" 171

> 		"disruption resistance" 0.3
> 		"disruption resistance energy" 6
> 		"disruption resistance heat" 3

Reviewer 2: ```suggestion
		"disruption resistance" 0.2
		"disruption resistance energy" 2
		"disruption resistance heat" 2
```

		angle 315
		back
	bay "Fighter" -167.5 -420
		angle 315
	bay "Fighter" 167.5 -420
		angle 45
	bay "Fighter" -179.5 80
		angle 45
		back
	bay "Fighter" 179.5 80
		angle 315
		back
	bay "Fighter" -167.5 -420
		angle 315
	bay "Fighter" 167.5 -420
		angle 45
	bay "Fighter" -179.5 80
		angle 45
		back
	bay "Fighter" 179.5 80
		angle 315
		back
	bay "Fighter" -167.5 -420
		angle 315
	bay "Fighter" 167.5 -420
		angle 45
	bay "Fighter" -179.5 80
		angle 45
		back
	bay "Fighter" 179.5 80
		angle 315
		back
	bay "Fighter" -167.5 -420
		angle 315
	bay "Fighter" 167.5 -420
		angle 45
	bay "Fighter" -179.5 80
		angle 45
		back
	bay "Fighter" 179.5 80
		angle 315
		back
	explode "tiny explosion" 30
	explode "small explosion" 45
	explode "medium explosion" 50
	explode "large explosion" 40
	explode "huge explosion" 50
	"final explode" "final explosion large"
	description "Korsmanath A'awojs, named in memory of the Korath home system, are massive nomadic habitats that hold much of their remaining population. True flying cities, they could be self-contained miniature worlds were it not for their heavy defensive armament and reliance on unsustainable materials, methods, and sources of power."

> 	description "	Commission of new Korsmanath A'awojs is an intensely resource-demanding, labor-intensive, painstaking process fraught with political intrigue. Navarchs fight with the civilian government for resources and yard time they would rather devote to building raiding and defensive fleets, civilians vie with each other for a place on the next ship, and planning committees argue with engineers over what methods and technologies to use. This leads to construction delays, corner-cutting, and over-militarization of the design at best, and violent feuds at worst."

Author: ```suggestion
	description "	Commission of new Korsmanath A'awojs is a resource-demanding, labor-intensive, painstaking process fraught with political intrigue. Navarchs fight with the civilian government for resources and yard time they would rather devote to building raiding and defensive fleets, civilians vie with each other for a place on the next ship, and planning committees argue with engineers over what methods and technologies to use. This leads to construction delays, corner-cutting, and over-militarization of the design at best, and violent feuds at worst."
```

		angle 70
	bay "Fighter" -167.5 -420
		angle 315
	bay "Fighter" 167.5 -420
		angle 45
	bay "Fighter" -179.5 80
		angle 45
		back
	bay "Fighter" 179.5 80
		angle 315
		back
	bay "Fighter" -167.5 -420
		angle 315
	bay "Fighter" 167.5 -420
		angle 45
	bay "Fighter" -179.5 80
		angle 45
		back
	bay "Fighter" 179.5 80
		angle 315
		back
	bay "Fighter" -167.5 -420
		angle 315
	bay "Fighter" 167.5 -420
		angle 45
	bay "Fighter" -179.5 80
		angle 45
		back
	bay "Fighter" 179.5 80
		angle 315
		back
	bay "Fighter" -167.5 -420
		angle 315
	bay "Fighter" 167.5 -420
		angle 45
	bay "Fighter" -179.5 80
		angle 45
		back
	bay "Fighter" 179.5 80
		angle 315
		back
	bay "Fighter" -167.5 -420
		angle 315
	bay "Fighter" 167.5 -420
		angle 45
	bay "Fighter" -179.5 80
		angle 45
		back
	bay "Fighter" 179.5 80
		angle 315
		back
	bay "Fighter" -167.5 -420
		angle 315
	bay "Fighter" 167.5 -420
		angle 45
	bay "Fighter" -179.5 80
		angle 45
		back
	bay "Fighter" 179.5 80
		angle 315
		back
	explode "tiny explosion" 30
	explode "small explosion" 45
	explode "medium explosion" 50
	explode "large explosion" 40
	explode "huge explosion" 50
	"final explode" "final explosion large"

> 	description "Korsmanath A'awojs, named in memory of the Korath home system, are massive nomadic habitats that hold much of their remaining population.  True flying cities, they could be self-contained miniature worlds were it not for their heavy defensive armament and reliance on unsustainable materials, methods, and sources of power."

Reviewer 5: ```suggestion
	description "Korsmanath A'awojs, named in memory of the Korath home system, are massive nomadic habitats that hold much of their remaining population. True flying cities, they could be self-contained miniature worlds were it not for their heavy defensive armament and reliance on unsustainable materials, methods, and sources of power."
```

> 	description "  Commission of new Korsmanath A'awojs is an intensely resource-demanding, labor-intensive, painstaking process fraught with political intrigue. Navarchs fight with the civilian government for resources and yard time they would rather devote to building raiding and defensive fleets, civilians vie with each other for a place on the next ship, and planning committees argue with engineers over what methods and technologies to use. This leads to construction delays, corner-cutting, and over-militarization of the design at best, and violent feuds at worst."

Reviewer 5: ```suggestion
	description "	Commission of new Korsmanath A'awojs is an intensely resource-demanding, labor-intensive, painstaking process fraught with political intrigue. Navarchs fight with the civilian government for resources and yard time they would rather devote to building raiding and defensive fleets, civilians vie with each other for a place on the next ship, and planning committees argue with engineers over what methods and technologies to use. This leads to construction delays, corner-cutting, and over-militarization of the design at best, and violent feuds at worst."
```

	"reverse engine" -31 -290
	"reverse engine" 31 -290
	engine -69 230
	engine 69 230
	turret 0 -166 "Grab-Strike Turret"
	turret -41 -69 "Grab-Strike Turret"
	turret 41 -69 "Warder Anti-Missile"
	turret -41.5 264 "Warder Anti-Missile"
	turret 41.5 264 "Banisher Grav-Turret"
	gun -75 -250 "Digger Mining Beam"
	gun 75 -250 "Digger Mining Beam"
	gun -60 -214.5 "Digger Mining Beam"
	gun 60 -214.5 "Digger Mining Beam"
	explode "tiny explosion" 30
	explode "small explosion" 45
	explode "medium explosion" 50
	explode "large explosion" 40
	explode "huge explosion" 50
	"final explode" "final explosion large"
	description `The largest of Korath production vessels are derived from a common and ancient chassis, including this large Efreti vessel. Facing very different challenges, and taking some inspiration from their Quarg protectors, the Efreti descendant of the "World-Ships" of old is lighter, faster, and more efficient, but less powerful than its Exile counterparts.`

ship "Arch-Carrack"
	sprite "ship/arch-carrack"
	thumbnail "thumbnail/arch-carrack"
	attributes
		category "Heavy Freighter"
		"cost" 15815000
		"shields" 21800
		"hull" 19300
		"required crew" 54
		"bunks" 97

> 		"mass" 1177

Reviewer 2: ```suggestion
		"mass" 1077
```

		"drag" 14.9
		"heat dissipation" .5
		"fuel capacity" 600
		"cargo space" 635
		"outfit space" 582
		"weapon capacity" 193
		"engine capacity" 147
		"disruption resistance" 0.24
		"disruption resistance energy" 2.4
		"disruption resistance heat" 2.4
		weapon
			"blast radius" 300
			"shield damage" 4600
			"hull damage" 2300
			"hit force" 5900
	outfits
		"Banisher Grav-Turret" 1
		"Digger Mining Beam" 2
		"Warder Anti-Missile" 2
		
		"Double Plasma Core"
		"Generator (Furnace Class)"
		"Systems Core (Large)"
		"Large Heat Shunt"
		"Fuel Processor"
		"Thermal Repeater Rifle" 55
		

> 		"Gaktem GP Hybrid Thruster"
> 		"Gaktem GP Hybrid Steering"

Reviewer 2: ```suggestion
		"Gaktem GP Hybrid Thruster"
		"Farves GP Hybrid Thruster"
		"Gaktem GP Hybrid Steering"
```

		"Scram Drive"

	engine 19.5 189.5
	engine -19.5 189.5
	engine 46 170 0.7
	engine -46 170 0.7
	"reverse engine" 14.5 -187.5
	"reverse engine" -14.5 -187.5
	turret 0 -119.5 "Warder Anti-Missile"
	turret 0 42 "Banisher Grav-Turret"
	turret 0 138.5 "Warder Anti-Missile"
	gun -37.5 -187 "Digger Mining Beam"
	gun 37.5 -187 "Digger Mining Beam"
	explode "tiny explosion" 120
	explode "small explosion" 60
	explode "medium explosion" 70
	explode "large explosion" 50
	explode "huge explosion" 15
	"final explode" "final explosion large"
	description "Although they do not effectively provide for their own defense, the Efreti remain responsible for the day-to-day needs of civilian and cargo transportation. While not as bulky or powerful as other Korath designs, this freighter is much faster at making runs back and forth between the Efreti's handful of remaining planets, while still holding enough defensive loadout to ward off the odd Mereti drone that slips past the Quarg."

ship "Charm-Shallop"
	sprite "ship/charm-shallop"
	thumbnail "thumbnail/charm-shallop"
	attributes
		category "Utility"

> 		"cost" 16570000

Reviewer 2: ```suggestion
		"cost" 14070000
```

		"shields" 19700
		"hull" 15200
		"required crew" 65

> 		"bunks" 253
> 		"mass" 575
> 		"drag" 8.5
> 		"heat dissipation" .6
> 		"fuel capacity" 500
> 		"cargo space" 194
> 		"outfit space" 569
> 		"weapon capacity" 153
> 		"engine capacity" 171

Reviewer 2: ```suggestion
		"bunks" 203
		"mass" 495
		"drag" 7.5
		"heat dissipation" .7
		"fuel capacity" 500
		"cargo space" 194
		"outfit space" 473
		"weapon capacity" 158
		"engine capacity" 129
```

		"disruption resistance" 0.2
		"disruption resistance energy" 2
		"disruption resistance heat" 2
		weapon
			"blast radius" 250
			"shield damage" 3600
			"hull damage" 1800
			"hit force" 5400
	outfits
		"Digger Mining Beam" 3
		"Warder Anti-Missile"
		"Banisher Grav-Turret"
		
		"Double Plasma Core"
		"Generator (Candle Class)"
		"Systems Core (Medium)"
		"Large Heat Shunt"
		"Fuel Processor"
		"Thermal Repeater Rifle" 66
		

> 		"Farves GP Hybrid Thruster"
> 		"Farves GP Hybrid Steering"

Reviewer 2: ```suggestion
		"Gaktem GP Hybrid Thruster"
		"Gaktem GP Hybrid Steering"
```
